app/gui2/src/components/GraphEditor.vue

```diff
@@ -457,6 +467,19 @@ async function readNodeFromClipboard() {
   )
 }
 
+async function readNodeFromExcelClipboard(htmlContent: string, clipboardItem: ClipboardItem): Promise<ClipboardData | undefined> {
+  // Check we have a valid HTML table
+  // If it is Excel, we should have a plain-text version of the table with tab separators.
+  if (clipboardItem.types.includes("text/plain") && htmlContent.startsWith("<table ") && htmlContent.endsWith("</table>")) {
+    const textData = await clipboardItem.getType("text/plain")
+    const text = await textData.text()
+    const payload = JSON.stringify(text).replaceAll(/^"|"$/g, "").replaceAll("'", "\\'")
```

Comment:
I guess it won't work well with cells containing `"`? Perhaps worth mentioning in a TODO.

Reply:
and/or opening an issue for it

Reply:
Works for those too.
![image](https://github.com/enso-org/enso/assets/4699705/1ae7da40-6c57-4cc1-a3fb-eabca2f41e7f)

Reply:
Looks like it should work with any input. The `replaceAll` regex here only removes the outermost `"` created by `stringify` on text. I guess that's good enough for now.

Reply:
note that we do have a function to escape enso text literals:

https://github.com/enso-org/enso/blob/62c93667084266d761928a2fb398bd3aea7df188/app/gui2/src/util/ast/text.ts#L16-L18

(note that it intentionally omits quotes so that it's suitable for use in interpolations)